code/modules/mob/living/living_defense.dm

```diff
@@ -257,24 +257,44 @@
 	spawn(1) updatehealth()
 	return 1
 
-/mob/living/proc/IgniteMob()
+/mob/living/proc/IgniteMob(var/fire_stacks_to_add = 0)

```

Comment:
You seem to have deprecated adding fire stacks manually. There are instances where we want to add fire stacks without igniting someone, for example smashing a bottle of flammable alcohol on them or getting welder fuel on them. Is there a reason you've replaced most instances of `adjust_fire_stacks()` to use `IgniteMob()`, and are you sure that this is the proper functionality in every case?

Reply:
Looks like every example where you replaced it also had `IgniteMob()` called.